config/locales/en.yml

```diff
@@ -113,91 +113,117 @@ en:
       heading: "There is a problem"
       radio_field: "Select %{field}"
       checkbox_field: "Select at least one %{field}"
-    questions:
-      urgent_medical_help:
-        title: "Do you need urgent medical help?"
-        description: "You can check if you have coronavirus (COVID-19) symptoms and get medical advice"
-        options:
-          - "Yes"
-          - "No"
+    groups:
+      help: # NHS question and 'What do you need help with'.
+        title: false # No title needed for this group.
```

Comment:
This there a risk that "false" could be displayed somewhere? Should this attribute be removed instead?

Reply:
Fixed - I've removed the key and made the default translation, which is used if there isn't a translation available, `false`.